src/test/java/core/basesyntax/OperationsTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax;
+
+import core.basesyntax.model.BalanceOperation;
+import core.basesyntax.model.PurchaseOperation;
+import core.basesyntax.model.ReturnOperation;
+import core.basesyntax.model.SupplyOperation;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class OperationsTest {
```

Comment:
Create a separate test class for each handler. Your test classes should have the same structure as the main solution.